src/main/kotlin/parts/wisdom/deansher/arcraider/Parts.kt

```diff
@@ -0,0 +1,259 @@
+@file:Suppress("PublicApiImplicitType")
```

Comment:
probably not a big deal for now, but I imagine we'll want to find a structure for splitting this up over multiple files.